Drivers/SensorFusion/Inc/sensor_fusion.hpp

```diff
@@ -14,8 +14,10 @@ extern "C"
 #ifndef SENSORFUSION_HPP
 #define SENSORFUSION_HPP
 
-#include "CommonDataTypes.hpp"
-#include "SF_Config.h"
+#include "common_data_types.hpp"
+#include "SF_config.h"
+#include "FreeRTOS.h"
+#include "semphr.h"
```

Comment:
Are these used in this file? We should only include the files we use in the file they are used in.

Reply:
Good point. I tested today. Both libraries have to be there because SemaphoreHandle_t is used in sensor_fusion.cpp.